src/interface/desktop/chat.html

```diff
@@ -10,6 +10,8 @@
     </head>
     <script type="text/javascript" src="./assets/purify.min.js?v={{ khoj_version }}"></script>
     <script type="text/javascript" src="./assets/markdown-it.min.js"></script>
+    <link rel="stylesheet" href="https://assets.khoj.dev/higlightjs/solarized-dark.min.css">
```

Comment:
Why is solarized dark css required here instead of just solarized?

Reply:
I like solarized dark 😬

Reply:
I'm going to merge this for now because public conversation references are broken, but we can come back to the stylesheet later. It's simple to update.